big_tests/tests/muc_SUITE.erl

```diff
@@ -319,16 +341,20 @@ init_per_suite(Config) ->
     %% For mocking with unnamed functions
     mongoose_helper:inject_module(?MODULE),
     Config2 = escalus:init_per_suite(Config),
-    Config3 = dynamic_modules:save_modules(domain(), Config2),
-    load_muc(muc_host()),
+    Config3 = dynamic_modules:save_modules(host_type(), Config2),
+    HostType = ct:get_config({hosts, mim, host_type}),
```

Comment:
You've just defined a helper `host_type()` function up there, could be defined in `domains_helper` and be used everywhere around big_tests btw.